create.sh

```diff
@@ -79,8 +81,7 @@ go run cmd/createorupdate/createorupdate.go
 
 az group deployment create -g $RESOURCEGROUP -n azuredeploy --template-file _data/_out/azuredeploy.json --no-wait
 
-# TODO(mjudeikis): after your DNS work these commands should move to the end of
-# this file.
+# This mimics RP dns record creation.
 hack/dns.sh zone-create $RESOURCEGROUP
```

Comment:
This is not at the end because healthcheck and sync in dev mode are blocking process and they need dns for checks (health check does)

Reply:
they should not be using the vanity dns for that -- by moving the dns creation the end, it'll prove that they're not ;)

Reply:
It will work for cases where PublicHostname is not provided and we default to FQDN as PublicHostname. Otherwise, health-check gets this as it tries to reach OCP console via external url. And this DNS does not exist at the moment. And I think this is expected behavior to check external url as it

```
+ go run cmd/healthcheck/healthcheck.go
panic: Head https://openshift.mjudeikis.osadev.cloud/console/: dial tcp: lookup openshift.mjudeikis.osadev.cloud on 127.0.0.1:53: no such host
```
Because of this: https://github.com/openshift/openshift-azure/blob/master/pkg/healthcheck/healthcheck.go#L87
But I think it does what it expected to do.

We could change it to use internal url only: 
`req, err := http.NewRequest("HEAD", "https://"+cs.Properties.MasterProfile.FQDN+"/console/", nil)`
cc: @kwoodson